File: mkblog-spring-data/src/main/java/com/mustafakoroglu/blog/mkblogspringdata/CustomerRepository.java
package com.mustafakoroglu.blog.mkblogspringdata;

import org.springframework.data.jpa.repository.JpaRepository;

import java.util.List;

/**
 * @author Mustafa Köroğlu <mustafa@koroglu.email>
 */
public interface CustomerRepository extends JpaRepository<Customer, Long> {
    List<Customer> findByAge(int age);
}

File: mkblog-thymeleaf-layout-dialect/src/main/kotlin/com/mustafakoroglu/blog/mkblogthymeleaflayoutdialect/repository/CustomerRepository.kt
package com.mustafakoroglu.blog.mkblogthymeleaflayoutdialect.repository

import com.mustafakoroglu.blog.mkblogthymeleaflayoutdialect.domain.Customer
import org.springframework.data.jpa.repository.JpaRepository

/**
 * @author Mustafa Köroğlu <mustafa@koroglu.email>
 */
interface CustomerRepository : JpaRepository<Customer, Long>